ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/search-settings/search-settings.tsx

```diff
@@ -0,0 +1,176 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+import * as React from 'react'
+import withListenTo, {
+  WithBackboneProps,
+} from '../../container/backbone-container'
+const user = require('../../../component/singletons/user-instance.js')
+const properties = require('../../../js/properties.js')
+const Property = require('../../../component/property/property.js')
+const PropertyView = require('../../../component/property/property.view.js')
+import MarionetteRegionContainer from '../../../react-component/container/marionette-region-container'
+const QuerySettingsView = require('../../../component/query-settings/query-settings.view.js')
+const QueryModel = require('../../../js/model/Query.js')
+const ConfirmationView = require('../../../component/confirmation/confirmation.view.js')
+import styled from '../../styles/styled-components'
+import { hot } from 'react-hot-loader'
+
+const Root = styled.div`
+  overflow: hidden;
+  padding: ${props => props.theme.minimumSpacing}
+    ${props => props.theme.minimumSpacing};
+`
+
+const PropertyResultCount = styled.div`
+  margin-bottom: ${props => props.theme.largeSpacing};
+`
+
+const QuerySettings = styled.div`
+  .editor-header,
```

Comment:
✏️ We should remove class selectors and use styled components instead

Reply:
.editor-header .editor-footer and editor-properties are within the marionette container. not sure what else to do in that case but individual styled components doesn't seem possible

Reply:
This is an interesting one - we don't really want to migrate these styles onto property view as they're only used here. This will be easy once `property.view` is converted as we can just use a `styled(Property)` extension.